packages/core/src/types/interfaces/i-document-data.ts

```diff
@@ -130,6 +130,11 @@ export interface IDocumentBody {
 
     customRanges?: ICustomRange[]; // plugin register，implement special logic for streams， hyperlink, field，structured document tags， bookmark，comment
     customDecorations?: ICustomDecoration[];
+
+    /**
+     * for copy/paste, data of custom-range and other module
+     */
+    payloads?: Record<string, string>;
```

Comment:
It would be saved to the disk?

Reply:
add an comment to mark it should not saved to disk